_playwright-tests/UI/CustomRepo.spec.ts

```diff
@@ -1,15 +1,15 @@
 import { test, expect, type Page } from '@playwright/test';
 import { navigateToRepositories } from './helpers/navHelpers';
 import { deleteAllRepos } from './helpers/deleteRepositories';
-import { closePopupsIfExist } from './helpers/helpers';
+import { closePopupsIfExist, retry } from './helpers/helpers';
```

Comment:
Although controversial, I agree with this :) I know that consoleDot sometimes gets stuck and I need to reload for it to load. But should we just not move this into the navigation function rather than introducing this to every single test? Would be cleaner and we would not forget to introduce it in other tests

Reply:
forgetting this can be an issue, but I didn't want to force it into existing helpers as I think it's mostly wanted only on the inital page load (and had some trouble trying this while prototyping) so I just added a new one, but it could be changed 😅 
@Andrewgdewar what's your opinion, retry included in nav helpers or it needs to be explicitly used when desired?

Reply:
I think moving it to the navigation functions makes sense, and keeping the helper around in case we create other navigation based things!

Reply:
changed!